### PR TITLE
chore: prepare for 0.17.0

### PR DESCRIPTION
**Pull request by CsCherrYY**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ All notable changes to the "vscode-java-dependency" extension will be documented
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 0.17.0
+### Added
+- Add contextual title and icon for `Java Project` explorer. [PR#423](https://github.com/microsoft/vscode-java-dependency/pull/423)
```
Comment (jdneo) [inline CHANGELOG.md:9]: Link to the issue if it exists.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,19 @@ All notable changes to the "vscode-java-dependency" extension will be documented
...
+- Allow adding library folder in `Java Project` explorer for unmanaged folders. [PR#434](https://github.com/microsoft/vscode-java-dependency/pull/434)
```
Comment (jdneo) [inline CHANGELOG.md:10]: Allow adding library folders into the `Referenced Libraries` on Windows and Linux (Press `Alt` to toggle out the button).

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,19 @@ All notable changes to the "vscode-java-dependency" extension will be documented
...
+
+### Changed
+- Make openFile experience consistent. [PR#426](https://github.com/microsoft/vscode-java-dependency/pull/426)
```
Comment (jdneo) [inline CHANGELOG.md:13]: Opening files from the `Java Project` explorer now has the same experience as the `File` explorer

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,19 @@ All notable changes to the "vscode-java-dependency" extension will be documented
...
+- Hide main classes from libraries in export jar wizard. [PR#428](https://github.com/microsoft/vscode-java-dependency/pull/428)
+- Hide view related commands in the command palette. [PR#439](https://github.com/microsoft/vscode-java-dependency/pull/439)
```
Comment (jdneo) [inline CHANGELOG.md:15]: Remove explorer related commands from the command palette.

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,19 @@ All notable changes to the "vscode-java-dependency" extension will be documented
...
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 0.17.0
+### Added
+- Add contextual title and icon for `Java Project` explorer. [PR#423](https://github.com/microsoft/vscode-java-dependency/pull/423)
+- Allow adding library folder in `Java Project` explorer for unmanaged folders. [PR#434](https://github.com/microsoft/vscode-java-dependency/pull/434)
+
+### Changed
+- Make openFile experience consistent. [PR#426](https://github.com/microsoft/vscode-java-dependency/pull/426)
+- Hide main classes from libraries in export jar wizard. [PR#428](https://github.com/microsoft/vscode-java-dependency/pull/428)
```
Comment (jdneo) [inline CHANGELOG.md:14]: Remove...from...

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,19 @@ All notable changes to the "vscode-java-dependency" extension will be documented
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 0.17.0
+### Added
+- Add contextual title and icon for `Java Project` explorer. [PR#423](https://github.com/microsoft/vscode-java-dependency/pull/423)
+- Allow adding library folder in `Java Project` explorer for unmanaged folders. [PR#434](https://github.com/microsoft/vscode-java-dependency/pull/434)
+
+### Changed
```
Comment (jdneo) [inline CHANGELOG.md:12]: One more item:

Add test metadata to the nodes which are under test source paths 

https://github.com/microsoft/vscode-java-dependency/pull/437

**Review by jdneo — CHANGES_REQUESTED** [review]

**Review by jdneo — APPROVED** [review]

**Comment by 0dinD** [comment]
@jdneo @CsCherrYY If it's not already too late (0.17.0 doesn't seem to be released yet as of writing), I think it would be a good idea to mention `Shift` + click as an alternative to `Alt` + click, because `Alt` + click won't work for the majority of Linux users. The icon for the alt command will show when pressing `Alt`, but the dialog won't open on click since the desktop environment captures that key/mouse combination for dragging windows ([see my comment to #434](https://github.com/microsoft/vscode-java-dependency/pull/434#issuecomment-756208089)).

**Comment by jdneo** [comment]
@0dinD Thanks for the reminder! @CsCherrYY Please add more description to the changelog when you have time.

**Pull request merged**